lua/nvim-treesitter/parsers.lua

```diff
@@ -2042,6 +2042,14 @@ list.slang = {
   experimental = true,
 }
 
+list.slim = {
+  install_info = {
+    url = "https://gitlab.com/theoreichel/tree-sitter-slim",
+    files = { "src/parser.c", "src/scanner.c" },
+  },
+  maintainers = { "@theoo", "@theoreichel" },
```

Comment:
That's the same person, right, just github and gitlab user names? If so, please remove the Gitlab username; the point of this information is to record who should be pinged on issues and PRs related to these queries(!), and that would obviously not work across code forges.